src/resmom/mom_comm.c

```diff
@@ -3337,13 +3339,15 @@ im_request(int stream, int version)
 
 #if	defined(MOM_CPUSET) && !defined(IRIX6_CPUSET)
 			if (new_cpuset(pjob) < 0) {
+				(void)mom_process_hooks(HOOK_EVENT_EXECJOB_ABORT, PBS_MOM_SERVICE_NAME, mom_host, &hook_input, &hook_output, hook_msg, sizeof(hook_msg), 1);
 				mom_deljob(pjob);
 				SEND_ERR(PBSE_SYSTEM)
 				goto done;
 			}
 #endif	/* MOM_CPUSET && !IRIX6_CPUSET */
 #if	MOM_BGL
 			if (verify_job_bgl_partition(pjob, NULL) != 0) {
+				(void)mom_process_hooks(HOOK_EVENT_EXECJOB_ABORT, PBS_MOM_SERVICE_NAME, mom_host, &hook_input, &hook_output, hook_msg, sizeof(hook_msg), 1);
```

Comment:
We should probably remove the BlueGene code rather than update it.

Reply:
True, I'll remove.